SConstruct

```diff
@@ -376,11 +376,13 @@ SConscript([
 ])
 if arch != "Darwin":
   SConscript([
-    'system/camerad/SConscript',
     'system/sensord/SConscript',
     'system/logcatd/SConscript',
   ])
 
+if arch == "larch64":
+  SConscript(['system/camerad/SConscript'])
```

Comment:
camerad doesn't run on pc, and will fail to build in release mode because the qcom camera stuff isn't copied